source/sass/blocks/bonus.scss

```
@@ -16,6 +16,10 @@
   margin-bottom: 33px;
   padding: 10px 61px 43px 58px;
 
+  @media (min-resolution: resolution2x) {
```

Так же как мы делали миксин для медиа по ширине, давай сделаем для ретины?

```
.bonus {
  ... 

  @include retina {
    ...
  }

  @include media-tablet {
    ...

    @include retina {
      ...
    }
  }
}
```

---

И переменную нам надо с `$` использовать, а то непонятно что это за странное `resolution2x` 🤔 

```suggestion
  @media (min-resolution: $resolution2x) {
```